src/UIComponents/Predict.jsx

```diff
@@ -51,12 +52,14 @@ class Predict extends Component {
                 <div style={styles.cardRow} key={index}>
                   <label>
                     {feature}
-                    : &nbsp;
+                    &nbsp;
                     <input
                       type="number"
                       onChange={event => this.handleChange(event, feature)}
                       value={this.props.testData[feature] || ""}
-                      placeholder={`min: ${+min}, max: ${+max}`}
+                      placeholder={I18n.t(
+                        "predictPlaceholder",
+                        {"minimum": min, "maximum": max})}
```

Comment:
I think we might want to keep the `+`s, per [this comment](https://github.com/code-dot-org/ml-playground/pull/122#discussion_r603569945).

Reply:
That's true... `min` is a fixed number so it would be printed with trailing zeroes to denote its significance. The `+` would essentially perform a `parseFloat` and convert it back to a float where it would be printed without them.

Reply:
fixed